src/Collection.php

```diff
@@ -201,15 +202,18 @@ public function aggregate(array $pipeline, array $options = [])
 
         $server = $this->manager->selectServer($options['readPreference']);
 
-        /* A "majority" read concern is not compatible with the $out stage, so
-         * avoid providing the Collection's read concern if it would conflict.
+        /* Servers before 4.2 don't support a readConcern when an $out stage is
+         * being used. Starting with 4.2, a read concern can be used; however it
+         * does not support the "linearizable" level. Using the "linearizable"
+         * level will cause a server error.
```

Comment:
We're not going to check for particular RC values here (like the old code did) so I think we can remove the text about "linearizable". I think this can just be:

> MongoDB 4.2 and later supports a read concern when an $out stage is being used, but earlier versions do not.

Reply:
Ah, yes. This was left over from a previous iteration where I assumed that we needed to avoid adding a `linearisable` level to the command. I realised later the spec actually expects us to pass it and let the server error. I'll update the comment as you suggested 👍